### ISSUE TITLE
How to import the code into Android studio?

### ISSUE THREAD

**Issue by DRFR33MAN24** [yoonusc/Android-POS-#2]
Hi. I'm having trouble importing the code into android studio, it seems that this project doesn't contain any manifest file or gradle scripts , it would be really great if you could include that into the project structure.

**Comment by nguyentthai96** [comment]
Hi I do not see the Android manifest and resource for android dev

**Comment by DRFR33MAN24** [comment]
yes same problem here years ago, It seems that the repo is only for checking code quality and it doesn't contain the  complete code, It's just for demo purposes apparently.
